src/main/java/edu/uz/dr/auctioneer/model/request/AuctionRequest.java

```diff
@@ -101,13 +98,17 @@ public void setUserDescription(final String userDescription) {
         this.userDescription = userDescription;
     }
 
+    public static String getDateFormat() {
```

Comment:
Ogólnie dopasowywanie kodu pod testy to nie jest za bardzo dobry pomysł. Poza tym to brzmi trochę bez sensu. Przecież to jest zmienna prywatna, to czemu w ogóle miałbyś testować dla niej gettery i settery? Nie rozumiem tego zbytnio. W takim wypadku pewnie powinno się dać jakoś to wykluczyć albo coś. Pojo tester jest po to, żeby testować istniejące metody, a nie po to, żeby dodawać nowe, niepotrzebne i wtedy je testować :D